cli/src/commands/new.rs

```diff
@@ -53,9 +53,9 @@ pub(crate) struct NewArgs {
         add = ArgValueCompleter::new(complete::revset_expression_all),
     )]
     revisions: Option<Vec<RevisionArg>>,
-    /// Ignored (but lets you pass `-o/-d`/`-r` for consistency with other
+    /// Ignored (but lets you pass `-o`/`-d`/`-r` for consistency with other
     /// commands)
-    #[arg(short = 'o', hide = true, short_aliases = ['d', 'r'],  action = clap::ArgAction::Count)]
+    #[arg(short = 'o', short_aliases = ['d', 'r'],  action = clap::ArgAction::Count)]
     unused_onto: u8,
```

Comment:
btw, the reason for this `u8` instead of `bool` is so you can repeat it (e.g. `jj new -r x -r y`). I suppose all the commands where positional arguments can be repeated (stored in a `Vec`) should use `u8`. That's mostly unrelated to this PR (or at least to this commit). If you don't fix it, I'll try to remember to do it once this PR has landed.

Reply:
https://github.com/jj-vcs/jj/pull/8343